source/common/http/filter_manager.h

```diff
@@ -138,6 +143,9 @@ struct ActiveStreamFilterBase : public virtual StreamFilterCallbacks,
   FilterManager& parent_;
   IterationState iteration_state_{};
 
+  const std::string custom_name_;
+  const std::string filter_name_;
```

Comment:
Probably better to just store the FactoryContext here? This would avoid the same general questions about custom  name vs. filter name which would be resolved by the comments in that struct.

Reply:
get it.